marketdata/rest.go

```diff
@@ -1234,6 +1239,108 @@ func (c *client) GetCryptoSnapshot(symbol string, exchange string) (*CryptoSnaps
 	return &snapshot, nil
 }
 
+// Sort represents the sort order of the results
+type Sort string
+
+// List of sort values
+var (
+	// SortDesc means the results will be sorted in a descending order
+	SortDesc Sort = "desc"
+	// SortAsc means the results will be sorted in an ascending order
+	SortAsc Sort = "asc"
+)
+
+// GetNewsParams contains optional parameters for getting crypto bars.
+type GetNewsParams struct {
+	// Symbols filters the news to the related symbols.
+	// If empty or nil, all articles will be returned.
+	Symbols []string
+	// Start is the inclusive beginning of the interval
+	Start time.Time
+	// End is the inclusive end of the interval
+	End time.Time
+	// Sort sets the sort order of the results. Sorting will be done by the UpdatedAt field.
+	Sort Sort
+	// IncludeContent tells the server to include the article content in the response.
+	IncludeContent bool
+	// ExcludeContentless tells the server to exclude articles that has no content.
+	ExcludeContentless bool
+	// TotalLimit is the limit of the total number of the returned news.
+	//
+	// If it's zero then the NoTotalLimit parameter is considered: if NoTotalLimit is true,
+	// then all the articles in the given start-end interval are returned.
+	// If NoTotalLimit is false, then 50 articles will be returned.
```

Comment:
I wouldn't mention 50 here, I would talk about using the maximum for one API call.

Reply:
Or is it 50 on purpose and not the default?

Reply:
There's no trivial way to use the server-side page limit maximum here, that's why I opted for this solution instead.